framework/src/main/java/org/tron/common/overlay/discover/node/statistics/NodeStatistics.java

```diff
@@ -31,8 +31,8 @@
   public final MessageStatistics messageStatistics = new MessageStatistics();
   public final MessageCount p2pHandShake = new MessageCount();
   public final MessageCount tcpFlow = new MessageCount();
-  public final SimpleStatter discoverMessageLatency;
-  public final SimpleStatter pingMessageLatency;
+  public final SimpleStarter discoverMessageLatency;
+  public final SimpleStarter pingMessageLatency;
```

Comment:
There is something wrong with your modification. Keep it as it is

Reply:
OK, had it reversed.